Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Source.jinja

```diff
@@ -324,14 +324,31 @@ void {{ ClassName }}::Set{{ UpperFirst(Property.attrib['Name']) }}(const {{ Prop
 {{     AutoComponentMacros.ParseRpcParams(Property, paramNames, paramTypes, paramDefines) }}
 void {{ ClassName }}::{{ UpperFirst(Property.attrib['Name']) }}({{ PrintRpcParameters('', paramDefines) }})
 {
+{% if (InvokeFrom == 'Authority') %}
+{#  InvokeFrom Authority inserts the invocation into the controller, not the component #}
+    if (!IsAuthority())
+{% elif (InvokeFrom == 'Autonomous') %}
+{#  InvokeFrom Authority inserts the invocation into the controller, not the component #}
+    if (!IsAutonomous())
```

Comment:
This really hammers home that these should be updated to IsNetEntityRoleAuthority/Autonomous

Reply:
Yup, consistency would be nice. Luckily Gene seems to have opened a pull-request for this just a few seconds ago 🤣